account_payment_return_import/wizard/payment_return_import.py

```diff
@@ -19,11 +19,11 @@ class PaymentReturnImport(models.TransientModel):
     _name = "payment.return.import"
     _description = "Import Payment Return"
 
-    @api.model
     def _compute_hide_journal_field(self):
         """ Return False if the journal_id can't be provided by the parsed
         file and must be provided by the wizard."""
-        return True
+        for record in self:
+            record.hide_journal_field = True
```

Comment:
In v13 an assignation over a whole recordset works:
```suggestion
        self.hide_journal_field = True
```

Reply:
I didn't know, thanks. It's already changed